recipes/libpcap/all/conanfile.py

```diff
@@ -115,6 +117,11 @@ def generate(self):
                 "--disable-dbus",
                 "--disable-rdma",
             ])
+            
+            snf = self.options.get_safe("with_snf")
+            if snf is not None:
+                tc.configure_args.append(f"--with-snf={yes_no(snf)}")
```

Comment:
```suggestion
            tc.configure_args.append(f"--with-snf={yes_no(self.options.with_snf)}")
```
Always present don't need to check.